worldedit-core/src/main/java/com/sk89q/worldedit/internal/Constants.java

```diff
@@ -35,11 +33,11 @@ private Constants() {
     public static final List<String> NO_COPY_ENTITY_NBT_FIELDS;
 
     static {
-        NO_COPY_ENTITY_NBT_FIELDS = Collections.unmodifiableList(Arrays.asList(
-                "UUIDLeast", "UUIDMost", "UUID", // Bukkit and Vanilla
-                "WorldUUIDLeast", "WorldUUIDMost", // Bukkit and Vanilla
-                "PersistentIDMSB", "PersistentIDLSB" // Forge
-        ));
+        NO_COPY_ENTITY_NBT_FIELDS = List.of(
```

Comment:
I wonder if we should change this to a `Set` and use `tag.getAllKeys().removeAll(Constants.NO_COPY_ENTITY_NBT_FIELDS);` at usage sites.

Reply:
I feel this would be better for a followup PR